package.json

```diff
@@ -26,7 +26,7 @@
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-sass-guidelines": "^8.0.0",
     "stylelint-prettier": "^1.1.2",
-    "typescript": "^4.4.3"
+    "typescript": "^4.9.5"
```

Comment:
The `package-lock.json` hasn't been updated (it's probably why the build is failing). An `npm i` should fix it.